menas/src/main/scala/za/co/absa/enceladus/menas/controllers/UserInfoController.scala

```diff
@@ -16,30 +16,27 @@
 package za.co.absa.enceladus.menas.controllers
 
 import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
-import org.springframework.beans.factory.annotation.Value
 import org.springframework.security.core.GrantedAuthority
 import org.springframework.security.core.context.SecurityContextHolder
 import org.springframework.security.core.userdetails.UserDetails
 import org.springframework.web.bind.annotation._
 import za.co.absa.enceladus.model.user.UserInfo
+import za.co.absa.enceladus.utils.general.ProjectMetadata
 
 @RestController
 @RequestMapping(Array("/api/user"))
-class UserInfoController extends BaseController {
-
-  @Value("${menas.version}")
-  val menasVersion: String = ""
+class UserInfoController extends BaseController with ProjectMetadata {
```

Comment:
You have both import and extends of ProjectMetadata

Reply:
Yes, you need to import it even if you extend it

Reply:
I thought I saw `._`